### ISSUE TITLE
filterPosts function multiselect

### ISSUE THREAD

**Issue by squaremunkeygist** [ivandoric/Theming-with-WP-REST-API#1]
How  would one transform the filterPosts function to work as a multiselect filter. ie using checkboxes to make multiple selections and select multiple  categories? Thanks.

**Comment by ivandoric** [comment]
You could try just changing radio buttons to checkboxes. And then in app.js file change `categoryFilter: ''` to `categoryFilter: []` so to make it accept an array.

**Comment by squaremunkeygist** [comment]
Thanks but it's not showing more then one filter. When i select two filters nothing shows. In the vue devtools the selected categories are showing up as an array but not on screen. Do i need to change something in the index for the array filter to work? thanks ivan

**Comment by ivandoric** [comment]
Yeah, doesn't work for me to. I think this functionallty doesn't come out of the box. You should probably write custom filter for something like that.

**Comment by squaremunkeygist** [comment]
I have come to the same conclusion but unfortunately my knowledge of vue.js is very limited. But how would one go about filtering posts with an array of id's. The id's are being tagged by vue.js ( see attached screenshot of vue devotees). Any thoughts are welcome.
![2016-06-01 16-54-53](https://cloud.githubusercontent.com/assets/3170079/15714131/9d30cec6-2819-11e6-8f21-2b8282ffb878.png)

**Comment by christianhamamdjian** [comment]
Have you found a solution for this problem yet?
